recipes/libtins/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "libtins" %}
+{% set version = "4.2" %}
+{% set sha256 = "a9fed73e13f06b06a4857d342bb30815fa8c359d00bd69547e567eecbbb4c3a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mfontanini/{{ name|lower }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libtins', max_pin='x') }}
+  skip: true  # [win and vc<14]
```

Comment:
We don't build for `vc<14` anymore, so this can be omitted.
```suggestion
```

Reply:
fixed.